Sources/SKSupport/ThreadSafeBox.swift

```diff
@@ -41,6 +41,11 @@ public class ThreadSafeBox<T>: @unchecked Sendable {
         _value = newValue
       }
     }
+    _modify {
+      lock.lock()
+      yield &_value
+      lock.unlock()
```

Comment:
Nit: best to use a `defer` here since the code after `yield` won't be executed if the caller throws, e.g `try x.value.someThrowingFn()`, `yield` is more like `try yield` in that case

Reply:
Oh, that’s subtle